tests/functional/reporting-non-auth.js

```diff
@@ -286,30 +268,23 @@ define(
           .end();
       },
 
-      "details param adds info to description": function() {
```

Comment:
> it seems like if I test the form submit (clicking submit buttons), this would be creating a new github issue? so I believed there was no practical way to test the newly requested behavior(??)

Since we control the entire stack, we can test it without creating new issues on GitHub. But yes, we don't want to *explicitly* test new issue creation, by creating actual new issues for the reasons you mentioned.

A few options would be to mock issue creation when the server is in test mode (via our `@mockable_response` decorator, or using the `mock` module, or possibly intercept all submit events, inspect what we need to, and cancel them via https://github.com/webcompat/webcompat.com/blob/master/tests/functional/lib/window-helpers.js.